test/graphql/mutations/associate_profiles_test.rb

```diff
@@ -15,6 +15,7 @@ class AssociateProfilesMutationTest < ActiveSupport::TestCase
   GRAPHQL
 
   setup do
+    PolicyHost.any_instance.stubs(:host_supported?).returns(true)
```

Comment:
When you know the `@host` instance, you can just mock the `:host_supported?` method on it directly. Calling the `any_instance` method is costly as it has to maintain a list of the instances and constantly traverse the object space.

Reply:
Okay, thanks! I didn't know I could call it on the `@host` instance because `host_supported?` is a `PolicyHost` method but I'll do that instead

Reply:
It didn't work with the `@host` instance and I think it might be for the reason that I said in the comment above? I think the instance that the test would need to use is the `PolicyHost` instance where host is the`@host' and policy is whatever policy is being used in the tests